components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/utils/APIExportUtil.java

```diff
@@ -535,6 +539,16 @@ private static void exportMetaInformation(String archivePath, API apiToReturn, R
             //If a web socket API is exported, it does not contain a swagger file.
             //Therefore swagger export is only required for REST or SOAP based APIs
             if (!APIConstants.APITransportType.WS.toString().equalsIgnoreCase(apiToReturn.getType())) {
+                //For Graphql APIs, the graphql schema definition is exported along with the swagger and api definition.
```

Comment:
Does this mean both swagger and graphql schema definition should be exported?

Reply:
yes we need both right?

Reply:
Yes. 
The commented section, 
//For Graphql APIs, the graphql schema definition is exported along **with the swagger and api definition** 
This is a bit confusing. Can you modify it?

Reply:
here the "API definition" refers to the api.json (serialized API object) in the exported API. will modify accordingly.